runtime/Makefile

```diff
@@ -25,9 +26,13 @@ test: all
 testCoverage: $(LIB) $(BCLIB)
 	$(MAKE) -C tests testCoverage
 
-$(LIB): $(OBJS)
+$(LIB): $(LIBBACKTRACE_OBJS) $(OBJS)
 	$(AR) r $@ $^
 
+$(LIBBACKTRACE_OBJS):
```

Comment:
I don't like this at all.

Why do we need a separate Makefile for libbacktrace? Use `$(addsuffix)` to generate the list of objects in the third-party/libbacktrace directory, and then use a static pattern rule to specify the compilation rule for those objects. The objects can then be added to balrt.a directly just as emyg_dtoa.o is.

Reply:
Changed to build libbacktrace without seperate MakeFile.